x-pack/packages/security/plugin_types_server/src/audit/index.ts

```diff
@@ -6,5 +6,10 @@
  */
 
 export type { AuditServiceSetup } from './audit_service';
-export type { AuditEvent, AuditHttp, AuditKibana, AuditRequest } from './audit_events';
-export type { AuditLogger } from './audit_logger';
+export type {
+  AuditEvent,
+  AuditHttp,
+  AuditKibana,
+  AuditLogger,
+  AuditRequest,
+} from '@kbn/core-security-server';
```

Comment:
re-exporting from the core security server so as not to have to update all imports